cloudhub/modules/ROOT/pages/_partials/ch-penetration-testing.adoc

```diff
@@ -0,0 +1,83 @@
+//Run Vulnerability Assessment and Penetration Tests
+
+//tag::intro[]
+If your company security policies require that you perform vulnerability and penetration testing against Anypoint Platform, you may commence testing without notifying Salesforce beforehand.
+
+All assessments must be performed in accordance with the https://help.salesforce.com/s/articleView?id=000392845&type=1[Security Assessment Agreement^].
+// end::intro[]
+
+//tag::runTest[]
+== Run Penetration and Vulnerability Tests
+
+Because penetration testing could interfere with other tenants, MuleSoft allows penetration testing on your workers but not on other Anypoint Platform services.
+
+[IMPORTANT]
+Salesforce does not address any vulnerabilities found with custom development.
+You must validate and fix any findings with your custom development.
+
+// end::runTest[]
+
+//tag::prereqIntro[]
+=== Prerequisites
+
+Before initiating penetration testing:
+// end::prereqIntro[]
+
+//tag::prereqCh1[]
+* Enable static IPs for the app.
++
+By default, CloudHub workers do not use static IP addresses, so you can't test them because their IP addresses might change. For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
```

Comment:
```suggestion
By default, CloudHub workers don't use static IP addresses, so you can't test them because their IP addresses might change. For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
```